tests/integration/api/controllers/users.spec.js

```diff
@@ -1616,6 +1615,43 @@ describe('Users API', () => {
           'contact.name': user.contact.name,
         });
       }
+
+      await Promise.all(savedUsers.map(async (savedUser) => {
```

Comment:
I think the probably deserves its own test case where we can be more intentional about writing users with different/same facility_id and contact_id values. (Really this is the only test that is going to validate interaction with the Couch view...) e.g.:

- User 1:
    - Facility A
    - Contact B
- User 2:
    - Facility A
    - Contact B
- User 3:
    - Facility A
    - Contact C
- User 4:
    - Facility D
    - Contact E

Then we can run several GET requests and make sure we get back what we expect:

- facility_id=A, contact_id=B
    - Returns 1,2
- facility_id=A
    - Returns 1,2,3
- contact_id=B
    - Returns 1,2

Reply:
I would like to add a scenario using a facility ID and/or contact ID that does not exist. What do you think?